Source/WebInspectorUI/UserInterface/Views/ComputedStyleSection.js

```diff
@@ -221,18 +240,28 @@ WI.ComputedStyleSection = class ComputedStyleSection extends WI.View
             expandableView.element.classList.add("computed-property-item");
             this.element.append(expandableView.element);
         }
+
+        if (this._filterText || this._filterCleared)
+            this.dispatchEventToListeners(WI.ComputedStyleSection.Event.FilterApplied, {matches: !this.isEmpty});
```

Comment:
is this actually necessary anymore now that we use `didLayoutSubtree`?

Reply:
It's not needed anymore in this implementation, but I'd like to keep it for now. 

It's wasteful that filtering triggers a section layout which blows away the entire list of properties. In a future patch, I want to optimize this so filtering should be a cheaper operation. Something like `WI.SpreadsheetCSSStyleDeclarationEditor.applyFilter()`. Then, dispatching the event will occur independent of layout.